internal/sqlitestore/team.go

```diff
@@ -0,0 +1,30 @@
+package sqlitestore
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/writhe/kudosaurus"
+)
+
+// Team - person model
+type Team struct {
+	gorm.Model
+	SlackID  string
+	Name     string
+	People   []Person `gorm:"foreignkey:TeamSlackID;association_foreignkey:SlackID"`
+	Settings Settings `gorm:"foreignkey:TeamSlackID;association_foreignkey:SlackID"`
+}
+
+func (t *Team) getData() kudosaurus.Team {
+	people := []kudosaurus.Person{}
```

Comment:
Just a small performance tip. You can preallocate such slices with 

`people := make([]kudosaurus.Person{},0,len(t.People))`